docs/docs/spark-procedures.md

```diff
@@ -936,3 +936,40 @@ as an `UPDATE_AFTER` image, resulting in the following pre/post update images:
 |-----|--------|--------------|
 | 3   | Robert | UPDATE_BEFORE|
 | 3   | Dan    | UPDATE_AFTER |
+
+## Table stats
+
+### `compute_table_stats`
+
+This procedure calculates the Number of Distinct Values (NDV) statistics for a specified table.
```

Comment:
What do you think to add a link to the blob type:  https://iceberg.apache.org/puffin-spec/#apache-datasketches-theta-v1-blob-type